docs/setup/install/auto-enroll.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="exclude"]
+
+If this is the first time you're starting {kib}, this command generates a 
+unique link to enroll your {kib} instance with {es}.
+
+. In your terminal, click the generated link to open {kib} in your browser.
+
+. In your browser, paste the enrollment token for {kib} and click the
+button to connect your {kib} instance with {es}.
```

Comment:
I think this sort of glosses over the difference between the Kibana auth code and the Elastic enrollment token. 

Maybe we could use a couple of screenshots here? Or maybe we could clarify this text a bit more? I'm not sure what the best approach is.

Reply:
So even if I go to `http://0.0.0.0:5601/` (without the unique part of the URL), Kibana prompts me to enter the enrollment token because it’s the first time I’ve configured it. That’s cool!

If I click **Configure manually** and then enter `http://0.0.0.0:9200`, Kibana indicates that it can't connect to my cluster.

![image](https://user-images.githubusercontent.com/25848033/142063758-e521b4a3-b078-405e-97bf-4bf9ad38099a.png)

If I then go back and enter the enrollment code that Elasticsearch generated, I get this prompt to enter a verification code. I looked back in the Kibana terminal, and sure enough, there was a verification code!

* @jportner, is this the behavior that you mentioned?
* @thomheymann, are users only prompted for this verification code if they attempt to configure manually? Also, do you know why I received an error when trying to connect to `http://0.0.0.0:9200` manually?

![image](https://user-images.githubusercontent.com/25848033/142063869-226576de-2d6e-4034-8041-6b0267fcfd7f.png)

Reply:
Joe's description is correct but I would call the code "verification code". We ask for it whenever someone attempts to configure Kibana using interactive setup UI without having followed the link that was generated when starting Kibana. It doesn't matter whether they do that using the enrollment token or using the manual configuration mode. Keep in mind that we only ask for the verification code when required (i.e. the user did not follow the interactive setup link and the user has entered the enrollment token / manual connection details and clicked "Configure Elastic") so you will not see it simply by viewing the enrollment token screen or the manual configuration screen.